src/module-elasticsuite-core/view/frontend/web/js/form-mini.js

```diff
@@ -206,7 +205,7 @@ define([
          * @private
          */
         _isEmpty : function(value) {
-            return value === null || value.trim().length === 0;
+            return value === null || value.trim().length < parseInt(this.options.minSearchLength, 10);
```

Comment:
Fallback to === 0; please so visibility of the fulltext search submit button is not impacted (we do not respect the min_query_length in fulltext search)